Building.md

```diff
@@ -210,36 +246,40 @@ on GitHub's UI.
 
 ### 4. Update `motoko-base`
 
-After releasing the compiler you can update `motoko-base`'s `master`
-branch to the `next-moc` branch.
+After releasing the compiler, update `motoko-base`'s `master` branch to the `next-moc` branch.
 
 * Wait ca. 5min after releasing to give the CI/CD pipeline time to upload the release artifacts
 * Change into `motoko-base` and pull the latest `next-moc`
 ```bash
 git switch next-moc; git pull
 ```
-* Create a new branch for the update
-```bash
-git switch -c $USER/update-moc-0.14.$MOC_MINOR
-```
 * Revise and update the `CHANGELOG.md`, by adding a top entry for the release
-* Update the `moc_version` env variable in `.github/workflows/{ci, package-set}.yml` and `mops.toml`
-  to the new released version:
-```bash
-perl -pi -e "s/moc_version: \"0\.14\.\\d+\"/moc_version: \"0.14.$MOC_MINOR\"/g; s/moc = \"0\.14\.\\d+\"/moc = \"0.14.$MOC_MINOR\"/g; s/version = \"0\.14\.\\d+\"/version = \"0.14.$MOC_MINOR\"/g" .github/workflows/ci.yml .github/workflows/package-set.yml mops.toml
-```
-* Add the changed files and commit the changes
+
+* Bump `moc` and create a PR:
 ```bash
-git add .github/ CHANGELOG.md mops.toml && git commit -m "Motoko 0.14."$MOC_MINOR
+# Create a new branch for the update
+git switch -c $USER/update-moc-0.14.$MOC_MINOR && \
+
+# Update the `moc_version` env variable in `.github/workflows/{ci, package-set}.yml` and `mops.toml` to the new released version
+perl -pi -e "s/moc_version: \"0\.14\.\\d+\"/moc_version: \"0.14.$MOC_MINOR\"/g; s/moc = \"0\.14\.\\d+\"/moc = \"0.14.$MOC_MINOR\"/g; s/version = \"0\.14\.\\d+\"/version = \"0.14.$MOC_MINOR\"/g" .github/workflows/ci.yml .github/workflows/package-set.yml mops.toml && \
+
+# Add the changed files and commit the changes
+git add .github/ CHANGELOG.md mops.toml && git commit -m "Motoko 0.14."$MOC_MINOR && \
+
+# Push the branch
+git push --set-upstream origin $USER/update-moc-0.14.$MOC_MINOR && \
```

Comment:
This is weird. You continue to a blank line which has no continuation...

Reply:
I've used the `&&` so that when one command fails for some reason the next ones will not be executed